app/assess/data.py

```diff
@@ -128,20 +132,38 @@ def get_round_with_applications(
     return None
 
 
+def get_score_and_justification(application_id, sub_criteria_id):
+    url = Config.ASSESSMENT_SCORES_ENDPOINT
+    params = {
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
+    }
+    response = requests.get(url, params)
```

Comment:
could we use `get_data` for this as above?

Reply:
using get data now 👍🏾